global_costmap:
  global_frame: map
  robot_base_frame: base_link
  update_frequency: 0.5
  static_map: true

  robot_radius: 0.4
  inflation_radius: 0.55

  plugins:
    - {name: static_map,       type: "costmap_2d::StaticLayer"}
    - {name: inflation_layer,  type: "costmap_2d::InflationLayer"}